Gemfile

```diff
@@ -26,56 +28,58 @@ group :doc do
 end
 
 # less support
+gem 'less-rails'
 gem 'therubyracer', platforms: :ruby
-gem "less-rails"
-gem "twitter-bootstrap-rails"
+gem 'twitter-bootstrap-rails'
 
-gem 'devise'
-gem 'devise_ichain_authenticatable', '>= 0.3.0'
+gem 'active_hash'
+gem 'axlsx_rails'
 gem 'cancancan', '~> 1.7'
-gem 'simple_form'
-gem 'show_for'
-gem 'inherited_resources'
-gem 'localized_country_select'
-gem "active_hash"
-gem 'cocoon'
-gem 'ransack'
-gem 'kaminari'
 gem 'carrierwave'
+gem 'clockwork'
+gem 'cocoon'
 gem 'daemons'
 gem 'date_validator'
+gem 'devise'
+gem 'devise_ichain_authenticatable', '>= 0.3.0'
 gem 'espinita'
+gem 'inherited_resources'
+gem 'kaminari'
+gem 'localized_country_select'
 gem 'prawn', '~> 0.13.0'
 gem 'prawn_rails'
-gem 'axlsx_rails'
-gem 'clockwork'
+gem 'ransack'
+gem 'show_for'
+gem 'simple_form'
```

Comment:
Did you reorder the gems or was that rubocop? And why did you change the order, if you did?